### ISSUE TITLE
Issue building Android project

### ISSUE THREAD

**Issue by erawhctim** [gojuno/mainframer#177]
First off - sorry that this isn't directly related to mainframer... but I'm hoping you can help me anyway.

Building my project both locally and remotely ssh'd into the remote machine works perfectly fine, however building through mainframer causes a build error relating to the "ANDROID_HOME" env. variable not being available. 

```
BUILD FAILED

Total time: 0.771 secs
SDK location not found. Define location with sdk.dir in the local.properties file or with an ANDROID_HOME environment variable.
```

...even though `echo $ANDROID_HOME` when ssh'd into the machine outputs the correct directory (the SDK is installed in `/usr/local/` and the env. var. is set globally for all users). 

Any idea how to get it working? I suppose I could manually add a`local.properties` file but that seems like the wrong solution (and might get tedious if we add multiple users into this). Any help is greatly appreciated!

**Comment by artem-zinnatullin** [comment]
Can you please check if https://github.com/gojuno/mainframer/issues/161 helps you?

**Comment by PaulWoitaschek** [comment]
I ended up with manually ssh-ing and creating a local.properties first.

**Comment by artem-zinnatullin** [comment]
Oh god :(

**Comment by ghostbuster91** [comment]
Maybe this will help: 
https://stackoverflow.com/questions/216202/why-does-an-ssh-remote-command-get-fewer-environment-variables-then-when-run-man

TLDR:
Neither `~/.bash_profile` nor  `~/.bash_login` nor  `~/.profile` are read when using bash in non-interactive mode. In non-interactive mode use the ssh environment variables.

`vi ~/.ssh/environment`
`VAR1=VALUE1`
`VAR2=VALUE2`

**Comment by dkonurov** [comment]
Can help this:
1) remote open /etc/enviroment
2) add on new line ANDROID_HOME = {FOLDER_ANDROID_SDK}
3) reboot remote machine

**Comment by artem-zinnatullin** [comment]
Closing for now, feel free to comment, we might reopen if it's something systematic (I haven't had issues with Android builds I've recently set up)
